### PR TITLE
FeatureIO for timestamp

### PR DESCRIPTION
**Pull request by jgersak**
add sub class for handling timestamps
add test

### CODE REVIEW

File: core/eolearn/core/eodata_io.py [hunk 1]
```diff
@@ -453,6 +454,14 @@ def _write_to_file(cls, data: _T, file: Union[BinaryIO, gzip.GzipFile], path: st
         file.write(json_data.encode())
 
 
+class FeatureIOTimeStamp(FeatureIOJson[_T]):
```
Comment (zigaLuksic) [inline core/eolearn/core/eodata_io.py:457]: This FeatureIO is specialized for `List[dt.datetime]` and should be marked as such. Just change both occurrences of `_T` with the mentioned type. This will correctly adjust the types of `save` and `load` methods.

Also we treat timestamp as a single word, therefore you should use `Timestamp` and not `TimeStamp`.

File: core/eolearn/tests/test_eodata_io.py [hunk 2]
```diff
@@ -314,6 +321,7 @@ def assert_data_equal(data1: Any, data2: Any) -> None:
             ),
         ),
         (FeatureIOJson, {"test": "test1", "test3": {"test": "test1"}}),
+        (FeatureIOTimeStamp, [datetime.datetime(2017, 1, 1, 10, 4, 7), datetime.datetime(2017, 1, 4, 10, 14, 5)]),
```
Comment (zigaLuksic) [inline core/eolearn/tests/test_eodata_io.py:324]: Do you think it would be worthwhile to add a test for an empty list? Perhaps one for an empty dictionary in FeatureIOJson as well...
